content/pages/de/jobs.rst

```diff
@@ -0,0 +1,31 @@
+Jobs
+#######
+
+:order: 2
+:date: 2021-04-02
+:icon: icon-device-laptop
+:summary: 
+:lang: de
+:slug: jobs
+
+.. container:: float-right
+
+    .. image:: {attach}/images/jobs/poetry-workshop.JPG
+        :width: 300px
+
+
+**Bist du auf der Suche nach einem Python Job?**
+
+Schau in unseren `Slack`_ in #jobs-europe Kanal und #feed-jobs (auf diesem Kanal findest du vor allem Jobs in den USA).
+
+
+
+**Hat deine Firma tolle Python Jobs die du teilen möchtest?**
+
+  - Veröffentliche den Job direkt in unserem `Slack`_ Kanal #job-europe.
+
+  - Für Job-Postings füllst du am besten dieses `Formular <http://bit.ly/pyladies-jobs-feed>`_ aus, die Anzeige wird dann automatisch 
+    im unserem #feed-jobs Slack Kanal veröffentlicht. 
```

Comment:
'in unserem'  or 'im' Slack Kanal ... but not 'im unserem'
Everything else is great!

Reply:
fixed